include/uapi/sound/sof/abi.h

```diff
@@ -19,14 +19,17 @@
  *    changes. PATCH should be reset to 0.
  *
  * 3) Increment PATCH version if you add backwards compatible bug fixes.
+ *
+ * All interface changes are agreed and documented in:
+ * https://github.com/orgs/thesofproject/projects/2
  */
 
 #ifndef __INCLUDE_UAPI_SOUND_SOF_ABI_H__
 #define __INCLUDE_UAPI_SOUND_SOF_ABI_H__
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 18
+#define SOF_ABI_MINOR 20
```

Comment:
Note: The kernel does not implement ALL changes of 3.20, so we cannot claim kernel support the full ABI of 3.20. E.g. missing: https://github.com/thesofproject/sof/pull/4808

And this is completely valid. We are not enforcing every firmware feature needs to be implemented by the kernel at the same time. But sometimes kernel features are done immediately (like the stop DMA).

So I think we should really drop this and go with https://github.com/thesofproject/linux/pull/3296